### ISSUE TITLE
PeriodicWildcardError/MissingInputException for imagenette2_train

### ISSUE THREAD

**Issue by zyzzyxdonta** [psteinb/sota_on_uncertainties#5]
I wrote a config for Slurm on hemera.
[config.txt](https://github.com/psteinb/sota_on_uncertainties/files/8462612/config.txt) (txt extension because GitHub 🙄)

Now if I run this:
```bash
snakemake -p --profile config/slurm/hemera imagenette2_train
```
I get this result:
```
Building DAG of jobs...
WorkflowError:
PeriodicWildcardError: The value .table in wildcard foldid is periodically repeated (00.table.table.table.table.table.table.table.table.table.table.table.table.table.table.table.table.table.table.table.table). This would lead to an infinite recursion. To avoid this, e.g. restrict the wildcards in this rule to certain values.
MissingInputException: Missing input files for rule create_tables:
    output: /home/pape58/Code/sota_on_uncertainties/data/imagenette2-320-splits/fold-00.table, /home/pape58/Code/sota_on_uncertainties/data/imagenette2-320-splits/fold-01.table, /home/pape58/Code/sota_on_uncertainties/data/imagenette2-320-splits/fold-02.table, /home/pape58/Code/sota_on_uncertainties/data/imagenette2-320-splits/fold-03.table, /home/pape58/Code/sota_on_uncertainties/data/imagenette2-320-splits/fold-04.table, /home/pape58/Code/sota_on_uncertainties/data/imagenette2-320-splits/fold-05.table, /home/pape58/Code/sota_on_uncertainties/data/imagenette2-320-splits/fold-06.table, /home/pape58/Code/sota_on_uncertainties/data/imagenette2-320-splits/fold-07.table, /home/pape58/Code/sota_on_uncertainties/data/imagenette2-320-splits/fold-08.table, /home/pape58/Code/sota_on_uncertainties/data/imagenette2-320-splits/fold-09.table, /home/pape58/Code/sota_on_uncertainties/data/imagenette2-320-splits/fold-10.table, /home/pape58/Code/sota_on_uncertainties/data/imagenette2-320-splits/fold-11.table, /home/pape58/Code/sota_on_uncertainties/data/imagenette2-320-splits/fold-12.table, /home/pape58/Code/sota_on_uncertainties/data/imagenette2-320-splits/fold-13.table, /home/pape58/Code/sota_on_uncertainties/data/imagenette2-320-splits/fold-14.table, /home/pape58/Code/sota_on_uncertainties/data/imagenette2-320-splits/fold-15.table, /home/pape58/Code/sota_on_uncertainties/data/imagenette2-320-splits/fold-16.table, /home/pape58/Code/sota_on_uncertainties/data/imagenette2-320-splits/fold-17.table, /home/pape58/Code/sota_on_uncertainties/data/imagenette2-320-splits/fold-18.table, /home/pape58/Code/sota_on_uncertainties/data/imagenette2-320-splits/fold-19.table
    wildcards: dataset=imagenette2-320
    affected files:
        /home/pape58/Code/sota_on_uncertainties/data/imagenette2-320-all
```

Am I doing something wrong?

**Comment by psteinb** [comment]
I'll look into it. But it will take some time.

**Comment by zyzzyxdonta** [comment]
OK, there's no hurry 😄

**Comment by psteinb** [comment]
I fixed that with decda67
